.storybook/webpack.config.js

```diff
@@ -73,5 +73,18 @@ module.exports = webpackSettings => {
     })
   );
 
+  config.module.rules.push({
+    test: /\.(ts|tsx)$/,
+    use: [
+      {
+        loader: require.resolve('babel-loader'),
+        options: {
+          presets: [['react-app', { flow: false, typescript: true }]],
+        },
+      },
+    ],
+  });
+  config.resolve.extensions.push('.ts', '.tsx');
```

Comment:
Let's just add these extensions to L32 (https://github.com/PocketDerm/radiance-ui/pull/148/files#diff-6da49b24efd584154cfb4ec759745192R32) and remove this line.

Reply:
done